ci-operator/step-registry/openshift-qe/ingress-perf/openshift-qe-ingress-perf-ref.yaml

```diff
@@ -0,0 +1,23 @@
+ref:
+  as: openshift-qe-ingress-perf
+  from_image:
+    namespace: ci
+    name: ocp-qe-perfscale-ci
+    tag: latest
+  commands: openshift-qe-ingress-perf-commands.sh
+  credentials:
+  - namespace: test-credentials # this entry injects the custom credential
+    name: ocp-qe-perfscale-es
+    mount_path: /secret
+  env:
+  - name: WORKLOAD_TYPE
```

Comment:
Since this commands script will only be running ingress perf tests I think we can hard code/export this in the commands script. 
It would be good to add maybe the config here so if we need to we can run different variations of the test

Reply:
Updated.